### PR TITLE
update README

### PR DESCRIPTION
**Pull request by fritzmg**
This actually mentions the Excel export and makes it more clear what you have to do, to enable the Excel export.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,12 +9,14 @@ same Lead.
 All configuration can be done in the form generator from Contao. Additionally you can set a label 
 for the backend module of your Lead and define the listing of the form data using simple tags.
 
-The leads extension offers additionally an export function (CSV and  - you need to install the
-"excel" extension) for each Lead in the backend. You can configure it as you wish!
+The leads extension additionally offers an export function for each Lead in the backend. You can
```
Comment (Toflar) [inline README.md]: The docs are not wrong. In the ER it's "excel" :-) Can you mention both, maybe? :-)

Comment (fritzmg) [inline README.md]: Hm, is that necessary? `contao-leads` is only available via composer anyway, isn't it? But sure, why not ;)

**Comment by Toflar** [comment]
First one afaik :)

**Comment by fritzmg** [comment]
done

**Comment by Toflar** [comment]
Cheers!

**Comment by fritzmg** [comment]
Sorry, there was a typo
```diff
-  [[php_exel](https://contao.org/en/extension-list/view/php_excel.en.html)]
+  [[php_excel](https://contao.org/en/extension-list/view/php_excel.en.html)]
```
Can you update it?

**Comment by Toflar** [comment]
https://github.com/terminal42/contao-leads/commit/ca59e8056abe335496274b28de8742566c85f87e

**Comment by fritzmg** [comment]
thx :)

**Pull request merged**